CONTRIBUTING.md

```diff
@@ -126,89 +126,3 @@ ___
 - Update the version of `bpmn-visualization` in [bpmnVisualization.yaml](inst/htmlwidgets/bpmnVisualization.yaml)
 ___
 
-## Maintainers
```

Comment:
I think it should be better to keep the title with the link of the new page, like it's done on [bpmn-visu](https://github.com/process-analytics/bpmn-visualization-js/blob/master/CONTRIBUTING.md#maintainers).

Reply:
Ok to do that in another PR